modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,51 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords: upgrade, promotion, premium, button
+
+== Premum upgrade promotion option
+
+{productname} 6.2 and later includes the `promotion` option. It controls the presentation or otherwise of a Tiny-specific promotion button.
+[[promotion]]
+
+== `+promotion+`
```

Comment:
```suggestion
{productname} 6.2 and later includes the `promotion` option. It controls the presentation or otherwise of a Tiny-specific promotion button.

[[promotion]]
== `+promotion+`
```

Reply:
Corrected as part of the ‘turn that section into a partial’ push just now.